src/main/java/com/gmail/luchyk/viktoriia/glovodb/service/OrderService.java

```diff
@@ -0,0 +1,148 @@
+package com.gmail.luchyk.viktoriia.glovodb.service;
+
+
+import com.gmail.luchyk.viktoriia.glovodb.converter.AddressConverter;
+import com.gmail.luchyk.viktoriia.glovodb.converter.CustomerConverter;
+import com.gmail.luchyk.viktoriia.glovodb.converter.OrderConverter;
+import com.gmail.luchyk.viktoriia.glovodb.converter.ProductConverter;
+import com.gmail.luchyk.viktoriia.glovodb.dto.Address;
+import com.gmail.luchyk.viktoriia.glovodb.dto.Customer;
+import com.gmail.luchyk.viktoriia.glovodb.dto.Order;
+import com.gmail.luchyk.viktoriia.glovodb.dto.Product;
+import com.gmail.luchyk.viktoriia.glovodb.entity.AddressEntity;
+import com.gmail.luchyk.viktoriia.glovodb.entity.CustomerEntity;
+import com.gmail.luchyk.viktoriia.glovodb.entity.OrderEntity;
+import com.gmail.luchyk.viktoriia.glovodb.entity.ProductEntity;
+import com.gmail.luchyk.viktoriia.glovodb.repository.AddressRepository;
+import com.gmail.luchyk.viktoriia.glovodb.repository.CustomerRepository;
+import com.gmail.luchyk.viktoriia.glovodb.repository.OrderRepository;
+import com.gmail.luchyk.viktoriia.glovodb.repository.ProductRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+
+@AllArgsConstructor
+@Service
+public class OrderService {
+    private OrderRepository orderRepository;
+    private ProductRepository productRepository;
+    private AddressRepository addressRepository;
+    private CustomerRepository customerRepository;
```

Comment:
I think it would be better to create separate services for each repository (they would perform some logic and communicate with the repo).